test/Microsoft.DotNet.Watcher.Tools.FunctionalTests/NoDepsAppTests.cs

```diff
@@ -79,6 +80,11 @@ public void RestartProcessThatTerminatesAfterFileChange()
                 // Then wait for it to restart when we change a file
                 using (var wait = new WaitForFileToChange(scenario.StartedFile))
                 {
+                    // On Unix the file write time is in 1s increments;
+                    // if we don't wait, there's a chance that the polling
+                    // watcher will not detect the change
+                    Thread.Sleep(1000);
```

Comment:
A second isn't that long. But is it possible to make this test poll? I'm hoping it can wait at most 1s (or perhaps 5s) but normally complete in a few nanoseconds.